pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -190,6 +275,19 @@ func formatAlias(query *cloudWatchQuery, stat string, dimensions map[string]stri
 		data[k] = v
 	}
 
+	/* CloudWatch labels for resource dimensions are space separated lists of the resource IDs.
+	 * A caller might want to access tags for any of the resources in the label, so we inject the tags with a unique prefix per part.
+	 */
+	for i, part := range strings.Split(label, " ") {
+		if _, ok := resourceTags[part]; !ok {
+			continue
+		}
+
+		for key, value := range resourceTags[part] {
+			data[fmt.Sprintf("tags.%d.%s", i, key)] = value
+		}
+	}
```

Comment:
It's more understandable to me if the variable is named `resource`, since it represents a resource ID. It's also more robust to use the default value in `resourceTags[resource]`, since it will correspond to an empty map:
```suggestion
	for i, resource := range strings.Split(label, " ") {
		tags := resourceTags[resource]
		for key, value := range tags {
			data[fmt.Sprintf("tags.%d.%s", i, key)] = value
		}
	}
```